sdk/suins-toolkit/.prettierrc.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Not sure why this still had its own custom config, but updated this to just use the same config as everything else